study.md

```diff
@@ -27,17 +27,17 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+<!-- The html written, and the view source is not the dom. The html is formated by your browser and turned into the dom. The view source just you the html, which is similar to the html you wrote, just compiled using a build process  -->
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+<!-- It is sorta, it is a visual representation of the dom. It is a compiling of different code and simplifies them so that it is easier to ready through. -->
```

Comment:
The DOM is an interface that represents HTML documents in a tree structure.  What you see in your web browser's developer tools is a visual representation of the browser's implementation of that interface, but not the DOM itself.